v0/pipeline/common/shell-script-inline-source.yaml

```diff
@@ -6,9 +6,7 @@ allOf:
     script:
       type: string
       metadata:
-        inputProperties:
-          type: string
-          internalType: shell_script
+        dependsOn: [ source.type ]
```

Comment:
@srdjan-harness currently the dependsOn field is also inside the inputProperties. For example you can check in JenkinsBuildStepInfo. And In BE also we are handling that structure only.

Reply:
If we want to move the dependsOn out of the inputProperties. Then we should for other steps schema as well for which we have the inside the inputProperties.